app/services/katello/cert/ca.rb

```diff
@@ -0,0 +1,21 @@
+module Katello
+  module Cert
+    module Ca
+      def self.ueber_cert(repository)
+        repository.organization.debug_cert
+      end
+
+      def self.ca_cert
+        File.open(Setting[:ssl_ca_file], 'r').read
+      end
+
+      def self.ssl_client_cert
+        OpenSSL::X509::Certificate.new(File.open(SETTINGS[:katello][:ssl_client_cert], 'r').read)
+      end
+
+      def self.ssl_client_key
+        OpenSSL::PKey::RSA.new(File.open(SETTINGS[:katello][:ssl_client_key], 'r').read)
+      end
```

Comment:
@jlsherrill I am trying to figure out how to organize this directory. I just stuck ssl_client_cert and ssl_client_key in here for now, but I think they should go in a separate module?